### ISSUE TITLE
Part of text shown in Russian

### ISSUE THREAD

**Issue by muldereric** [seansfkelley/synology-download-manager#124]
Firefox extension version 0.6.0
DSM version 6.2.2-24922 Update 3 
Download station version 3.8.14-3543
No debug info available.

Issue at hand:
I have the add-on installed in Firefox 69.0.1 64-bit edition. Work great, but since the latest update a part of the text shown in the task window is in Russian while the rest is in English (like it should be). See the screen capture below.
I tried reinstalling the add-on, but it didn't help.

![Task overview with Russian text](https://user-images.githubusercontent.com/10549960/66079613-af2d9080-e564-11e9-94f8-b6798d23e96a.png)

**Comment by seansfkelley** [comment]
I did recently add support for Russian, but it's very strange that it should be appearing for you while everything else is English. Did you ever install a Russian language pack, or switch your computer to Russian, or otherwise do anything that might have stuck around and cause the extension to pick the wrong language?

One specific thing you can do to help me debug, if you don't mind, would be:

1. go to `about:debugging`
2. scroll down to Synology Download Manager and hit the Inspect button
3. in the page that opens, go to the Console tab (second from the left on top)
4. you'll probably see a ton of messages already there, but you can ignore those 
5. for each of the three following things, can you type them in at the bottom text field and hit enter and record the answer?
    - `browser.i18n.getUILanguage()`
    - `browser.i18n.getMessage('@@ui_locale')`
    - `browser.i18n.getAcceptLanguages().then(r => console.log(r))`

Those should hopefully narrow down if some parts of the browser think your languages are configured differently than others.

**Comment by Shralldam** [comment]
Hello, I'm encountering the same problem. I went ahead and used the console to retrieve the data you asked for:

```
browser.i18n.getUILanguage()
"fr"
```
```
browser.i18n.getMessage('@@ui_locale')
"fr"
```
```
browser.i18n.getAcceptLanguages().then(r => console.log(r))
Promise { <state>: "pending" }

Array(4) [ "fr", "fr-fr", "en-us", "en" ]
```

Hope this helps,

Damien

**Comment by seansfkelley** [comment]
Awesome, thank you @Shralldam for that information. The problem is that I was testing English and Russian configurations, but the bug only happened for languages that were neither, so I was on the wrong track. Good thing you speak French!

This should be fixed and now released as version 0.6.1.

**Comment by SpiReCZ** [comment]
Same here:

```
browser.i18n.getUILanguage()
"cs"
```

```
browser.i18n.getMessage('@@ui_locale')
"cs"
```

```
browser.i18n.getAcceptLanguages().then(r => console.log(r))
Promise { <state>: "pending" }

Array(4) [ "cs", "sk", "en-US", "en" ]
```
